### PR TITLE
feat(be): Updates to AWSSecurityHub notifier for creds

### PR DESCRIPTION
**Pull request by charmik-redhat**
## Description

Same as title

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

Manual testing

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: central/notifiers/awssh/notifier.go [hunk 1]
```diff
@@ -167,15 +193,42 @@ func newNotifier(configuration configuration) (*notifier, error) {
...
 
+func getCredentials(config configuration) (*storage.AWSSecurityHub_Credentials, error) {
+	if !env.EncNotifierCreds.BooleanSetting() {
+		return config.descriptor.GetAwsSecurityHub().GetCredentials(), nil
+	}
+
+	if config.descriptor.GetNotifierSecret() == "" {
+		return nil, errors.Errorf("encrypted notifier credentials for notifier '%s' empty", config.descriptor.GetName())
+	}
+	if config.cryptoCodec == nil {
```
Comment (clickboo) [inline central/notifiers/awssh/notifier.go:213]: this and the next if check might be overkill

File: central/notifiers/email/email.go [hunk 2]
```diff
@@ -552,6 +546,16 @@ func (e *email) getPassword() (string, error) {
 		return e.creds, nil
 	}
 
+	if e.notifier.GetNotifierSecret() == "" {
+		return "", errors.Errorf("encrypted notifier credentials for notifier '%s' empty", e.notifier.GetName())
+	}
+	if e.cryptoCodec == nil {
```
Comment (clickboo) [inline central/notifiers/email/email.go:552]: same as above

**Review by clickboo — APPROVED** [review]

File: central/notifiers/awssh/notifier.go (hunk 1 above)
Comment (charmik-redhat) [reply central/notifiers/awssh/notifier.go:213]: Removed the empty check for key, but I think we can keep this one in-case someone called the NewEmail and similar funs with a nil value for codec.

File: central/notifiers/email/email.go (hunk 2 above)
Comment (charmik-redhat) [reply central/notifiers/email/email.go:552]: Removed the empty check for key, but I think we can keep this one in-case someone called the NewEmail and similar funs with a nil value for codec.

**Pull request merged**